packages/theme-preset/src/create-preset.test.tsx

```diff
@@ -0,0 +1,38 @@
+import { defaultTheme } from '@marigold/system';
+import { createPreset } from './create-preset';
+
+test('create preset and merge it with defaultTheme', () => {
```

Comment:
Maybe another tests that shows that `preset.content` finds the closest marigold path?

Reply:
Tried to but i can have a closer look